index.js

```diff
@@ -323,7 +312,11 @@ async function apostrophe(options, telemetry, rootSpan) {
     if (self.taskRan) {
       await self._exit();
     } else {
-      await self.emit('run', self.isTask());
+      const SIGNAL = { exit: null };
```

Comment:
This isn't a simple constant, so the uppercase feels inappropriate. The name is also a bit confusing. Its purpose is to decide what happens after, so let's call it `after`.